dev-ruby/async/async-1.30.1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 Gentoo Authors
```

Comment:
Why did you add this old slot? It doesn't look like it is needed for any of the packages in this PR, so it will be obsolete from the start. But perhaps I'm overlooking something?